client/my-sites/checkout/src/components/wp-checkout.tsx

```diff
@@ -776,6 +797,64 @@ const CheckoutSummaryBody = styled.div`
 	}
 `;
 
+const CheckoutTermsAndCheckboxesWrapper = styled.div`
+	display: flex;
+	flex-direction: column;
+	padding-top: 32px;
+	padding-left: 40px;
```

Comment:
The padding regression is introduced here, I think essentially you're adding padding to the left which pushes the ToS and line items off the page on mobile devices. This can probably be fixed with a media query like:

```suggestion
	padding: 32px 0 0 40px;
	@include breakpoint-deprecated( "<660px" ) {
        padding: 1em;
```

Reply:
Nice find. I think I fixed it in 1f9d72b0b4a.